pkg/operation/botanist/component/nginxingress/nginxingress.go

```diff
@@ -368,3 +595,22 @@ func getLabels(resourceType string) map[string]string {
 
 	return labels
 }
+
+func (n *nginxIngress) getArgs(configMap *corev1.ConfigMap) []string {
+	ingressClass := "seed-nginx"
+	if version.ConstraintK8sGreaterEqual122.Check(n.values.KubernetesVersion) {
+		ingressClass = "k8s.io/seed-nginx"
+	}
+
+	return []string{
+		"/nginx-ingress-controller",
+		"--default-backend-service=garden/nginx-ingress-k8s-backend",
+		"--enable-ssl-passthrough=true",
+		"--publish-service=garden/nginx-ingress-controller",
+		"--election-id=ingress-controller-seed-leader",
+		`--ingress-class=` + ingressClass,
+		"--update-status=true",
+		"--annotations-prefix=nginx.ingress.kubernetes.io",
+		`--configmap=garden/` + configMap.Name,
+	}
```

Comment:
I don't see the 

```
            {{- if semverCompare ">= 1.22-0" .Capabilities.KubeVersion.GitVersion }}
            - --controller-class={{ include "nginx-ingress.class" . }}
            {{- end }}
```

part which was in the Helm charts. Do I miss something?

Reply:
I added it later in another commit gardener@973ae11